bundles/org.openhab.binding.serial/src/main/java/org/openhab/binding/serial/internal/channel/ChannelConfig.java

```diff
@@ -73,4 +74,14 @@ public class ChannelConfig {
      * Decrease value
      */
     public @Nullable String decreaseValue;
+
+    /**
+     * Command for refesh command
+     */
+    public @Nullable String refreshValue;
+
+    /**
+     * Automatic refresh interval
+     */
+    public int refreshInterval = 0;
```

Comment:
Should this really be 0 as default?

Reply:
An automatic refresh is only required if the peer does not actively push value changes but requires, for example, a periodic “status” command to send the current values.

Nevertheless, the value of “Refresh Values” is independent because this is also required for querying the initial value after the connection.

I will add a clarification into the thing-types.xml and ChannelConfig.java.